Auto

Rylan-Meilutis opened the pull request:
Working auto

Rylan-Meilutis committed:
fixing rev stuff

Rylan-Meilutis committed:
not good yet

Rylan-Meilutis committed:
fixes

DevilTech committed:
added spikes and limit switches for climber

DevilTech committed:
fix ports

JackDunfey commented on src/main/java/frc/robot/subsystems/Climber.java:
```diff
@@ -56,8 +66,8 @@ public Climber(OperatorInterface oi) {
         climber.configSupplyCurrentLimit(climberLimit, TIMEOUT);
-        climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
-        climber.configReverseLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
+        // climber.configForwardLimitSwitchSource(LimitSwitchSource.FeedbackConnector, LimitSwitchNormal.NormallyOpen, 0);
```
Why were these removed?

mmarchetti replied:
The limit switches aren't compatible with the Falcon, so we are implementing the limit switch checks ourselves.

JackDunfey replied:
Oh ok

JordanCieslinski committed:
Updated DIO ports for limit switches

DevilTech committed:
Lots of Auto Work. 2 ball auto is really close

mmarchetti committed:
shooter calibration

Rylan-Meilutis committed:
removed unused stuff

the @suppresswarnings in visionCOntrol.java is for the final boolean record, so there is no dead code warning

DevilTech committed:
e
